tests/homeScreening.spec.js

```diff
@@ -0,0 +1,70 @@
+import { jest } from '@jest/globals';
+import getTenScreenings from '../src/getTenScreenings.js';
+
+describe('getRecentReviews()', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.clearAllTimers();
+  });
+
+  test('Only show screenings for the next five days', async () => {
+    jest.setSystemTime(new Date('2024-02-02T13:37:00.000Z'));
+
+    const getTenScreeningsAdapter = {
+      loadAllScreenings: async () => [
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-10T19:00:00.000Z' }),
+      ],
+    };
+
+    const data = await getTenScreenings(getTenScreeningsAdapter);
+    expect(data).toHaveLength(2);
+  });
+
+  test('Only show screenings for the next five days', async () => {
```

Comment:
The name of this test is identical to the previous one, so it would be more difficult to understand in the terminal. Maybe the name could have been elaborated to explain the difference between this test and the last one.